### ISSUE TITLE
v0.5.0-alpha.0 controller trying to pull  cdi-importer:v0.6.0-alpha.0 ?

### ISSUE THREAD

**Issue by shiywang** [kubevirt/containerized-data-importer#213]
```
[cnv-qe-jenkins@cnv-executor-shiywang ~]$ oc get pods -o yaml
apiVersion: v1
items:
- apiVersion: v1
  kind: Pod
  metadata:
    annotations:
      openshift.io/scc: restricted
    creationTimestamp: 2018-06-15T00:04:54Z
    generateName: cdi-deployment-775d5bb9bd-
    labels:
      app: cdi-controller
      pod-template-hash: "3318166568"
    name: cdi-deployment-775d5bb9bd-8sb2d
    namespace: golden-images
    ownerReferences:
    - apiVersion: extensions/v1beta1
      blockOwnerDeletion: true
      controller: true
      kind: ReplicaSet
      name: cdi-deployment-775d5bb9bd
      uid: 4e5c0bdb-6faf-11e8-97e1-fa163ecad8f5
    resourceVersion: "161723"
    selfLink: /api/v1/namespaces/golden-images/pods/cdi-deployment-775d5bb9bd-8sb2d
    uid: bb703840-702f-11e8-97e1-fa163ecad8f5
  spec:
    containers:
    - env:
      - name: OWN_NAMESPACE
        valueFrom:
          fieldRef:
            apiVersion: v1
            fieldPath: metadata.namespace
      image: kubevirt/cdi-controller:v0.5.0-alpha.0
      imagePullPolicy: IfNotPresent
      name: cdi-controller
      resources: {}
      securityContext:
        capabilities:
          drop:
          - KILL
          - MKNOD
          - SETGID
          - SETUID
        runAsUser: 1000130000
      terminationMessagePath: /dev/termination-log
      terminationMessagePolicy: File
      volumeMounts:
      - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
        name: cdi-sa-token-l2x85
        readOnly: true
    dnsPolicy: ClusterFirst
    imagePullSecrets:
    - name: cdi-sa-dockercfg-r6p4f
    nodeName: cnv-executor-shiywang-node1.example.com
    restartPolicy: Always
    schedulerName: default-scheduler
    securityContext:
      fsGroup: 1000130000
      seLinuxOptions:
        level: s0:c11,c10
    serviceAccount: cdi-sa
    serviceAccountName: cdi-sa
    terminationGracePeriodSeconds: 30
    volumes:
    - name: cdi-sa-token-l2x85
      secret:
        defaultMode: 420
        secretName: cdi-sa-token-l2x85
  status:
    conditions:
    - lastProbeTime: null
      lastTransitionTime: 2018-06-15T00:04:54Z
      status: "True"
      type: Initialized
    - lastProbeTime: null
      lastTransitionTime: 2018-06-15T00:05:00Z
      status: "True"
      type: Ready
    - lastProbeTime: null
      lastTransitionTime: 2018-06-15T00:04:54Z
      status: "True"
      type: PodScheduled
    containerStatuses:
    - containerID: cri-o://967c3b19fbf6a4e9ffb21f2db180cff243d009ced33f6a7e7c1f335a582b2645
      image: docker.io/kubevirt/cdi-controller:v0.5.0-alpha.0
      imageID: docker.io/kubevirt/cdi-controller@sha256:1d07ee6ff1083855d14e76be48e122cfd5ecd426b1d4c4441dc724fdcf8a666e
      lastState: {}
      name: cdi-controller
      ready: true
      restartCount: 0
      state:
        running:
          startedAt: 2018-06-15T00:05:00Z
    hostIP: 172.16.0.24
    phase: Running
    podIP: 10.129.0.20
    qosClass: BestEffort
    startTime: 2018-06-15T00:04:54Z
- apiVersion: v1
  kind: Pod
  metadata:
    annotations:
      cdi.io/storage.createdByController: "yes"
      openshift.io/scc: restricted
    creationTimestamp: 2018-06-15T00:05:19Z
    generateName: importer-golden-pvc-
    labels:
      app: containerized-data-importer
    name: importer-golden-pvc-78jgd
    namespace: golden-images
    resourceVersion: "162894"
    selfLink: /api/v1/namespaces/golden-images/pods/importer-golden-pvc-78jgd
    uid: ca34a2ba-702f-11e8-97e1-fa163ecad8f5
  spec:
    containers:
    - args:
      - -v=1
      env:
      - name: IMPORTER_ENDPOINT
        value: https://ajo.es/binaries/cirros-0.4.0-x86_64-disk.img
      image: docker.io/kubevirt/cdi-importer:v0.6.0-alpha.0
      imagePullPolicy: IfNotPresent
      name: importer
      resources: {}
      securityContext:
        capabilities:
          drop:
          - KILL
          - MKNOD
          - SETGID
          - SETUID
        runAsUser: 1000130000
      terminationMessagePath: /dev/termination-log
      terminationMessagePolicy: File
      volumeMounts:
      - mountPath: /data
        name: cdi-data-vol
      - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
        name: default-token-hfhzq
        readOnly: true
    dnsPolicy: ClusterFirst
    imagePullSecrets:
    - name: default-dockercfg-9w7gk
    nodeName: cnv-executor-shiywang-master1.example.com
    restartPolicy: Never
    schedulerName: default-scheduler
    securityContext:
      fsGroup: 1000130000
      seLinuxOptions:
        level: s0:c11,c10
    serviceAccount: default
    serviceAccountName: default
    terminationGracePeriodSeconds: 30
    volumes:
    - name: cdi-data-vol
      persistentVolumeClaim:
        claimName: golden-pvc
    - name: default-token-hfhzq
      secret:
        defaultMode: 420
        secretName: default-token-hfhzq
  status:
    conditions:
    - lastProbeTime: null
      lastTransitionTime: 2018-06-15T00:05:50Z
      status: "True"
      type: Initialized
    - lastProbeTime: null
      lastTransitionTime: 2018-06-15T00:05:50Z
      message: 'containers with unready status: [importer]'
      reason: ContainersNotReady
      status: "False"
      type: Ready
    - lastProbeTime: null
      lastTransitionTime: 2018-06-15T00:05:50Z
      status: "True"
      type: PodScheduled
    containerStatuses:
    - image: docker.io/kubevirt/cdi-importer:v0.6.0-alpha.0
      imageID: ""
      lastState: {}
      name: importer
      ready: false
      restartCount: 0
      state:
        waiting:
          message: Back-off pulling image "docker.io/kubevirt/cdi-importer:v0.6.0-alpha.0"
          reason: ImagePullBackOff
    hostIP: 172.16.0.21
    phase: Pending
    podIP: 10.128.0.97
    qosClass: BestEffort
    startTime: 2018-06-15T00:05:50Z
kind: List
metadata:
  resourceVersion: ""
  selfLink: ""
```
```
[cnv-qe-jenkins@cnv-executor-shiywang ~]$ oc get pods
NAME                              READY     STATUS         RESTARTS   AGE
cdi-deployment-775d5bb9bd-8sb2d   1/1       Running        0          3m
importer-golden-pvc-78jgd         0/1       ErrImagePull   0          2m
[cnv-qe-jenkins@cnv-executor-shiywang ~]$ oc get vms
NAME      AGE
test-vm   49m
[cnv-qe-jenkins@cnv-executor-shiywang ~]$ oc describe pod importer-golden-pvc-78jgd
Name:         importer-golden-pvc-78jgd
Namespace:    golden-images
Node:         cnv-executor-shiywang-master1.example.com/172.16.0.21
Start Time:   Fri, 15 Jun 2018 00:05:50 +0000
Labels:       app=containerized-data-importer
Annotations:  cdi.io/storage.createdByController=yes
              openshift.io/scc=restricted
Status:       Pending
IP:           
Containers:
  importer:
    Container ID:  
    Image:         docker.io/kubevirt/cdi-importer:v0.6.0-alpha.0
    Image ID:      
    Port:          <none>
    Args:
      -v=1
    State:          Waiting
      Reason:       ErrImagePull
    Ready:          False
    Restart Count:  0
    Environment:
      IMPORTER_ENDPOINT:  https://ajo.es/binaries/cirros-0.4.0-x86_64-disk.img
    Mounts:
      /data from cdi-data-vol (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from default-token-hfhzq (ro)
Conditions:
  Type           Status
  Initialized    True 
  Ready          False 
  PodScheduled   True 
Volumes:
  cdi-data-vol:
    Type:       PersistentVolumeClaim (a reference to a PersistentVolumeClaim in the same namespace)
    ClaimName:  golden-pvc
    ReadOnly:   false
  default-token-hfhzq:
    Type:        Secret (a volume populated by a Secret)
    SecretName:  default-token-hfhzq
    Optional:    false
QoS Class:       BestEffort
Node-Selectors:  <none>
Tolerations:     <none>
Events:
  Type     Reason            Age               From                                                Message
  ----     ------            ----              ----                                                -------
  Warning  FailedScheduling  2m (x6 over 3m)   default-scheduler                                   pod has unbound PersistentVolumeClaims (repeated 3 times)
  Normal   Scheduled         2m                default-scheduler                                   Successfully assigned importer-golden-pvc-78jgd to cnv-executor-shiywang-master1.example.com
  Normal   Pulling           2m                kubelet, cnv-executor-shiywang-master1.example.com  pulling image "docker.io/kubevirt/cdi-importer:v0.6.0-alpha.0"
  Warning  Failed            2m                kubelet, cnv-executor-shiywang-master1.example.com  Failed to pull image "docker.io/kubevirt/cdi-importer:v0.6.0-alpha.0": rpc error: code = Unknown desc = manifest unknown: manifest unknown
  Warning  Failed            2m                kubelet, cnv-executor-shiywang-master1.example.com  Error: ErrImagePull
  Normal   SandboxChanged    1m (x22 over 2m)  kubelet, cnv-executor-shiywang-master1.example.com  Pod sandbox changed, it will be killed and re-created.
```

**Comment by copejon** [comment]
@shiywang As part of converting CDI to tracking it's own versions (instead of mirroring kubevirt's versions) we have released v0.5.0 and v1.0.0.  `*-alpha` versions should NOT be used as they are considered broken.

v0.5.0 reflects the legacy PVC annotations and v1.0.0 reflects the new PVC annotations.

**Comment by nellyc** [comment]
this was caused by the release cycle issue.
fixed by the 0.5.0 release
closing
